src/parser/classes/menus/MultiPageMenuSection.ts

```diff
@@ -0,0 +1,16 @@
+import Parser from '../../index.js';
```

Comment:
Please used named imports for the Parser instead of a default one, please see #535 for why. You only need to change the files you added, the linked pull request changes all other ones.
```suggestion
import { Parser } from '../../index.js';
```